src/main/java/com/orbitz/consul/monitoring/ClientEventCallback.java

```diff
@@ -6,12 +6,30 @@
 
 public interface ClientEventCallback {
 
+    /**
+     * @deprecated use {@link #onHttpRequestSuccess(String, String, String, String, int)} instead.
+     */
+    @Deprecated(forRemoval = true)
```

Comment:
Is is ok to use Java 9 style @Deprecated in this project?

Reply:
Deleted forRemoval attribute